pkgs/development/python-modules/openmm/default.nix

```diff
@@ -0,0 +1,87 @@
+{ buildPythonPackage, lib, fetchFromGitHub, cmake, clang, gfortran, fftwSinglePrec
+, doxygen, ocl-icd, opencl-headers, swig, python, cython, numpy
+, cudatoolkit, enableCuda ? false
+} :
+
+buildPythonPackage rec {
+  pname = "openmm";
+  version = "7.7.0";
+
+  src = fetchFromGitHub {
+    owner = "openmm";
+    repo = pname;
+    rev = version;
+    hash = "sha256-2PYUGTMVQ5qVDeeABrwR45U3JIgo2xMXKlD6da7y3Dw=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    gfortran
+    swig
+    cython
+    doxygen
+  ];
+
+  buildInputs = [
+    fftwSinglePrec
+    ocl-icd
+    opencl-headers
+  ] ++ lib.optional enableCuda cudatoolkit
+  ;
+
+  propagatedBuildInputs = [
+    python
+    numpy
+  ];
+
+  dontUseSetuptoolsBuild = true;
+  dontUsePipInstall = true;
+  dontUseSetuptoolsCheck = true;
+
+  cmakeFlags = [
+    "-DCMAKE_C_COMPILER=${clang}/bin/clang"
+    "-DCMAKE_CXX_COMPILER=${clang}/bin/clang++"
+    "-DBUILD_TESTING=ON"
+    "-DOPENMM_BUILD_AMOEBA_PLUGIN=ON"
+    "-DOPENMM_BUILD_CPU_LIB=ON"
+    "-DOPENMM_BUILD_C_AND_FORTRAN_WRAPPERS=ON"
+    "-DOPENMM_BUILD_DRUDE_PLUGIN=ON"
+    "-DOPENMM_BUILD_PME_PLUGIN=ON"
+    "-DOPENMM_BUILD_PYTHON_WRAPPERS=ON"
```

Comment:
I'm assuming this provides a Python 2 API. Since Python 2 is deprecated, can we hide that behind an option?

Reply:
Is purely python3 now.